index.src.html

```diff
@@ -640,6 +642,35 @@ <h3 id="user-mediation">
 
   ISSUE: Bring in some of felt@'s ideas here.
 
+  <h3 id="sanitize-data">
+    Sanitize the data handled in the feature
+  </h3>
+
+  It is always a good strategy to consider the kinds of data a new feature
+  is processing. For example, new features allowing the readout of data may
+  want to adopt specific privacy strategies such as minimizing the quality
+  of datas (quantization) or reducing the frequency, in line with standard
+  privacy engineering practices. Examples
+  *   [BATTERY-STATUS-API] <em>“The user agent should not expose high
+    precision readouts”</em>
+  * [SENSORS-API] <em>“Limit maximum sampling frequency”, “Reduce accuracy”</em>
```

Comment:
Are there any more (possibly)?

Reply:
I've filed an issue to track finding more for a future version: https://github.com/w3cping/security-questionnaire/issues/16